jupyterlab_git/handlers.py

```diff
@@ -473,7 +473,8 @@ async def post(self, path: str = ""):
         """
         data = self.get_json_body()
         commit_msg = data["commit_msg"]
-        body = await self.git.commit(commit_msg, self.url2localpath(path))
+        amend = data["amend"]
```

Comment:
We could make this one optional to be more forgiven:

```suggestion
        amend = data.get("amend", False)
```